modulus/experimental/models/sfno/initialization.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import math
+import torch
+import warnings
+
+
+def _no_grad_trunc_normal_(tensor, mean, std, a, b):
```

Comment:
A lot of these are duplicates of what is in [`afno/distributed/layers.py`](https://github.com/NVIDIA/modulus/blob/main/modulus/models/afno/distributed/layers.py). Should remove this redundancy at some point.

Reply:
I agree. I think we will pursue this in a separate PR for a proper integration of SFNO into Modulus

Reply:
Created an issue to track this: https://github.com/NVIDIA/modulus/issues/173